README.md

```diff
@@ -1,31 +1,28 @@
 # Revenge
 
 A modification for the Discord Android app. Continuation of [Vendetta](https://github.com/vendetta-mod). Get it? Vendetta is like, revenging..., yeah.
+> [!WARNING]
+> Revenge is still WIP! Please understand that you might encounter some issues.
 
 ## ⬇️ Installing
 
 > [!NOTE]
-> While Revenge is platform-agnostic, there is no guarantee it will always work on iOS. I do not have a device to test the changes I've made, so feel free to report iOS-specific issues in the Issues tab if you encounter one.
-
-1. Install [Vendetta](https://github.com/vendetta-mod/Vendetta)
-2. Go to **Settings** > **General** enable **Developer Settings**
-3. Go to **Settings** > **Developer**, enable **Load from custom URL**, and input
-   ```
-   https://cdn.jsdelivr.net/gh/revenge-mod/builds@main/revenge.js
-   ```
-   in the new input box labelled **VENDETTA URL**.
-   - If the **Developer** settings section does not appear, restart the app
-5. Restart the app and check if it says Revenge in the settings
+> While Revenge is platform-agnostic, there is no guarantee it will always work on iOS. I do not have a device to test the changes I've made, so feel free to report iOS-specific issues if you encounter one.
 
+1. Install [Vendetta Manager](https://github.com/vendetta-mod/VendettaManager/releases/latest/download/Manager.apk)
+2. Download (don't install!) [latest Xposed module](https://github.com/revenge-mod/RevengeXposed/releases/latest/download/app-release.apk)
+3. Go to Vendetta manager's **Settings > About** and press version number **10 times**
+4. Set the custom Xposed module location to *`/storage/emulated/0/Download/app-release.apk`*
```

Comment:
Bold and also capitalize the setting name here:

```
Set the **Custom Xposed module location** [...]
```

Also use a codeblock for the path so it's easily copy-able.

What happens if the user downloads it to somewhere else other than that path? should probably include some text above saying this is the default path?